packages/ui/primitives/stepper/step.tsx

```diff
@@ -0,0 +1,133 @@
+import type { ReactNode } from 'react';
```

Comment:
This might have specialised a little too much into our specific UI, let's try and keep it a little more headless so we could use it for other kinds of forms such as team creation 😄